hdr_schemata/models/GWDM/v1_1/Organisation.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+from pydantic import AnyUrl, BaseModel, EmailStr, Field, constr
+from hdr_schemata.definitions.HDRUK import *
+
+
+class Organisation(BaseModel):
+    name: Optional[Name] = Field(
```

Comment:
Is this field changing from `publisherName` to `name`?  If `publisherName` is hardcoded in places like the api is it going to need updating there?

Reply:
Yep it is indeed, hence I'll have many PRs to come 😢

Reply:
Ok fair enough. It's early days of the GWDM at the moment but for the future I'd say a field name change like that would constitute a breaking change, given that it requires code changes.

Reply:
yup for sure. Spoke with Loki about this yesterday and we hope that a change like this really wont be needed again, as you say, we get away with it now as it's early days 👍